src/processors/modulation/scanner_vibrato/ScannerVibrato.cpp

```diff
@@ -167,7 +183,15 @@ void ScannerVibrato::processAudio (AudioBuffer<float>& buffer)
 
             auto* outData = audioOutBuffer.getWritePointer (ch);
             for (int i = 0; i < ScannerVibratoWDF::numTaps; ++i)
+            {
+                if (stereoMode && ch == 1)
+                {
+                    FloatVectorOperations::negate (modMixData[i], modMixData[i], numSamples);
```

Comment:
This is how flipping the phase was done in the phaser and tremolo modules, but is it a good way to achieve the flip in this module where the setup is a little different.

Reply:
Yeah, `modMixData` should always be in the range [0, 1] at this point, so this should be correct!